Model/ResourceModel/CsvHsCode.php

```diff
@@ -27,16 +27,28 @@ protected function _construct()
         $this->_init('reach_hs_code', 'id');
     }
 
+    public function testProductDetails($sku) {
+        $connection = $this->getConnection();
+        $select = $connection->select()->from($this->getMainTable());
+        $select->where('sku = ?', $sku);
+        $result = $connection->fetchRow($select);
+        return $result;
+    }
+
     public function getHsCode($sku)
     {
         $connection = $this->getConnection();
         $select = $connection->select()->from($this->getMainTable());
         $select->where('sku = ?', $sku);
         $result = $connection->fetchRow($select);
-        if (count($result) && isset($result['hs_code'])) {
-            return $result['hs_code'];
+
+        if ($result) {
```

Comment:
like you mentioned this is the same change that Suraiya made in the MAG-81 code review. Once we merge MAG-81 could you get that change into your branch and remove these changes so there's not a conflict? (I don't know if that's the right process but that's what I've got)

Reply:
is this the fix to be able to add varient items to the cart? This is the only tangible code change I can find other than debug statements (of course I could 100% be blind)

Reply:
Yes, this fixes the varients.

Reply:
And yes, I can merge the two changes together.

Reply:
I think putting extra check on `result` is already in MAG-81. If checking `result` is the only change in this branch in the exact same area/file then this branch can be deprecated altogether without any extra work I think . Is there anything else changed here that I am missing but would be needed for the fix of the issue with variant? Thanks!

Reply:
I think we can toss this entire branch. @lizzynibali ?